src/FixedDataTable.js

```diff
@@ -346,7 +351,7 @@ var FixedDataTable = React.createClass({
   },
 
   _shouldHandleWheelX(/*number*/ delta) /*boolean*/ {
-    if (this.props.overflowX === 'hidden') {
+    if (this.props.overflowX === 'hidden' || this.props.disableWheelHandler) {
```

Comment:
why? ReactWheelHandler takes both `_shouldHandleWheel*` functions as parameters, and since it is initialised in `componentWillMount`, changing `disableWheelHandler` during the lifecycle of the component wouldn't change any behaviour.